assets/rust analyzer.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
oh, let's rename the file to `rust-analyzer.svg`

Whitespace in file names is weird :)

otherwise, lgtm!

bors d+

Reply:
Fixed.